ndk/src/events/content-tagger.ts

```diff
@@ -163,7 +163,7 @@ export async function generateContentTags(
     await Promise.all(promises);
 
     content = content.replace(hashtagRegex, (tag, word) => {
-        const t: NDKTag = ["t", word];
+        const t: NDKTag = ["t", word.slice(1)];
```

Comment:
This is the change that fixes the hashtag bug

Because of the change to the hashtag regex, we were getting `["t", "#hello"]` instead of `["t", "hello"]`.